src/Header/CdrEofHeader.php

```diff
@@ -0,0 +1,71 @@
+<?php
+declare(strict_types = 1);
+
+namespace ZipStream\Header;
+
+/**
+ * Class CdrEofHeader
+ * @package ZipStream\Header
+ */
+class CdrEofHeader
+{
+    use HeaderHelperTrait;
+
+    /**
+     * @var int
+     */
+    private $offset;
+
+    /**
+     * @var int
+     */
+    private $headerOffset;
+
+    /**
+     * @var int
+     */
+    private $fileCount;
+
+    /**
+     * CdrEofHeader constructor.
+     * @param int $offset
+     * @param int $headerOffset
+     * @param int $fileCount
+     */
+    public function __construct($offset, $headerOffset, $fileCount)
+    {
+        $this->offset = $offset;
+        $this->headerOffset = $headerOffset;
+        $this->fileCount = $fileCount;
+    }
+
+    /**
+     * Send CDR EOF (Central Directory Record End-of-File) record.
+     *
+     * @param resource $outputStream
+     * @param string   $comment
+     */
+    public function write($outputStream, string $comment)
```

Comment:
I feel like writing doesn't belong to a Header. Seems to me a responsibility of a 'StreamWriter' kind of class. I'd prob replace it with 'generate()' and return whatever it needs to be written. I think this would also help in case we want to test the generated bytes, e.g, in case we want to make sure the right header signature is included. Thoughts?